.changeset/olive-comics-knock.md

```diff
@@ -0,0 +1,37 @@
+---
+"@comet/cms-api": minor
+"comet-docs": minor
+---
+
+Add AccessLog Interceptor where request data is logged in the standard console. The access log allows to see who accessed what data and when. This information for example can be used to identify security vulnerabilities.
```

Comment:
```suggestion
Add AccessLog Interceptor to log information about the request to standard output. The informations contains information about the requester and the request itself. This can be useful for fulfilling legal requirements regarding data integrity or for forensics.
```

Reply:
```suggestion
Add access logging to log information about the request to standard output. The log contains information about the requester and the request itself. This can be useful for fulfilling legal requirements regarding data integrity or for forensics.
```

The usage of an interceptor is an implementation detail IMO